docs/reference/faq.mdx

```diff
@@ -0,0 +1,81 @@
+
+Our goal is to make distributed applications easier, so developers and operators can be more productive.
+
+### What is included in the wasmCloud ecosystem?
+
+wasmCloud provides all the tools to build, deploy, and manage complex distributed applications. In addition to the [host](/docs/concepts/hosts), wasmCloud also has a [CLI](/docs/cli/), a [UI](/docs/cli/ui), and a [deployment manager](/docs/ecosystem/wadm/).
+
+### Is wasmCloud Open Source?
+
+Yes! wasmCloud is a [CNCF Sandbox project](https://www.cncf.io/projects/wasmcloud/). wasmCloud has applied for [Incubating status](https://github.com/cncf/toc/pull/1198).
+
+The source code for wasmCloud is available on [GitHub](https://github.com/wasmCloud), under [Version 2 of the Apache License](https://github.com/wasmCloud/wasmCloud/blob/main/LICENSE).
+
+### Is wasmCloud a WebAssembly runtime?
+
+Not exactly. We sometimes refer to wasmCloud [hosts](/docs/concepts/hosts) as "application runtimes." Hosts do run [WebAssembly components](/docs/concepts/webassembly-components), through the use of [wasmtime](https://wasmtime.dev).
+
+### What is the unit of compute for wasmCloud?
+
+wasmCloud applications are composed of [actors](/docs/concepts/actors) and [capability providers](/docs/concepts/capabilities), which are [linked](/docs/concepts/runtime-linking) together at runtime. Decoupling business logic from capabilities is part of the design philosophy of wasmCloud.
+
+### I see wasmCloud actors are stateless, but I need to store state. Is wasmCloud right for me?
+
+wasmCloud applications can access and store state via [capability providers](/docs/concepts/capabilities). Only actors are stateless.
+
+### Do capability providers implement capabilities, or do they act as proxies to external capabilities?
+
+That depends on the specific capability provider. For example, the [HTTP Server provider](https://github.com/wasmCloud/wasmCloud/tree/main/crates/providers/http-server) _is_ a server, while the [Redis provider](https://github.com/wasmCloud/wasmCloud/tree/main/crates/providers/kv-redis) acts as a client to an external Redis server.
+
+### How does wasmCloud handle secrets?
+
+There are several kinds of secrets involved in distributed applications. Configuration for actors and providers can contain sensitive data. Today, these values are associated with [runtime links](/docs/concepts/runtime-linking), or given to providers [on startup](/docs/cli/ctl#options-3). The host itself is also configured with sensitive data, which are passed to the host [on startup](/docs/reference/host-config).
```

Comment:
Pending a secrets management RFC, I think it's worth calling out application level secrets.
```suggestion
There are several kinds of secrets involved in distributed applications. Configuration for actors and providers can contain sensitive data. Today, these values are associated with [runtime links](/docs/concepts/runtime-linking), or given to providers [on startup](/docs/cli/ctl#options-3). The host itself is also configured with sensitive data, which are passed to the host [on startup](/docs/reference/host-config).

If you're working with secrets in your application, it's recommended to use a capability provider like [kv-vault](https://github.com/wasmCloud/capability-providers/tree/main/kv-vault) that proxies to a secrets management solution.
```